include/pinocchio/algorithm/rnea.hxx

```diff
@@ -99,8 +99,7 @@ namespace pinocchio
 
         const JointIndex i = jmodel.id();
         const JointIndex parent = model.parents[i];
-
-        jmodel.jointVelocitySelector(data.tau) = jdata.S().transpose() * data.f[i];
+        jmodel.jointVelocityExtendedModelSelector(data.tau) += jdata.S().transpose() * data.f[i];
```

Comment:
Add a comment to explain the += allow to manage mimic joint (the quantity added to mimicked joint)